src/azure-cli/azure/cli/command_modules/servicefabric/_help.py

```diff
@@ -464,6 +464,15 @@
         az sf managed-cluster client-certificate delete -g testRG -c testCluster --common-name Contoso.com
 """
 
+helps['sf managed-cluster network-security-rule add'] = """
+type: command
+short-summary: Add a network security rule to a manged cluster.
+examples:
+  - Add network security rule.
+  - text: >
+        az sf managed-cluster network-security-rule add -g testRG -c testCluster --name 'network security rule name' --access allow --description 'network security rule description' --direction inbound --protocol tcp --priority 1200 --sourcePortRanges ''1-1000' --destinationPortRanges '1-65535' --sourceAddressPrefixes '167.220.242.0/27 167.220.0.0/23 131.107.132.16/28 167.220.81.128/26' --destinationAddressPrefixes '194.69.104.0/25 194.69.119.64/26 167.220.249.128/26 255.255.255.255/32'
```

Comment:
addressprefix params should be space separated stings instead of a string with space separated prefixes no? sfmc expects an array of strings, and when you specify these parameters with nargs="+" that means that you can pass entries like this:
--sourceAddressPrefixes '167.220.242.0/27'  '167.220.0.0/23'  '131.107.132.16/28'  '167.220.81.128/26'

Reply:
for some reason this does not work. I keep getting unrecognized arguments error below: 
ERROR    cli.azure.cli.core.azclierror:azlogging.py:212 unrecognized arguments: --source_port_ranges 1-10001122-65535 --destination_port_ranges 1-19002200-65535 --source_address_prefixes 167.220.242.0/27167.220.0.0/23131.107.132.16/28167.220.81.128/26 --destination_address_prefixes 194.69.104.0/25194.69.119.64/26167.220.249.128/26255.255.255.255/32
ERROR    az_command_data_logger:azlogging.py:213 unrecognized arguments: --source_port_ranges 1-10001122-65535 --destination_port_ranges 1-19002200-65535 --source_address_prefixes 167.220.242.0/27167.220.0.0/23131.107.132.16/28167.220.81.128/26 --destination_address_prefixes 194.69.104.0/25194.69.119.64/26167.220.249.128/26255.255.255.255/32
---------- generated xml file: C:\Users\mwesigwaguma\.azdev\env_config\repos\azure-cli\env\test_results.xml -----------
=============================================== short test summary info ===============================================
FAILED src\azure-cli\azure\cli\command_modules\servicefabric\tests\latest\test_sf_managed_cluster.py::ServiceFabricManagedClustersTests::test_network_security_rule - SystemExit: 2

Reply:
I instead used  action='append' instead of nargs=+
it was working before with all the values in a string but im not sure what I changed.